03/03/2021 01:09:55|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 01:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 01:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 01:09:55|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
03/03/2021 01:42:07|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 01:52, because of: KeyError. Exception info: 'season'|
03/03/2021 01:42:09|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 01:52, because of: KeyError. Exception info: 'season'|
03/03/2021 01:42:18|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
03/03/2021 04:00:01|INFO    |root                            |BAZARR All existing episode subtitles indexed from disk.|
03/03/2021 04:09:55|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 04:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 04:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 04:09:55|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
03/03/2021 04:41:59|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 04:51, because of: KeyError. Exception info: 'season'|
03/03/2021 04:42:01|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 04:52, because of: KeyError. Exception info: 'season'|
03/03/2021 04:42:10|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
03/03/2021 05:00:00|INFO    |root                            |BAZARR All existing movie subtitles indexed from disk.|
03/03/2021 06:43:40|INFO    |root                            |BAZARR Running cache maintenance|
03/03/2021 07:01:32|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 07:06:55|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 07:16, because of: KeyError. Exception info: 'season'|
03/03/2021 07:07:23|INFO    |root                            |BAZARR Finished searching for Subtitles to upgrade. Check History for more information.|
03/03/2021 07:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 07:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 07:09:55|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
03/03/2021 07:41:21|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 07:42:10|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 07:52, because of: KeyError. Exception info: 'season'|
03/03/2021 07:42:13|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 07:52, because of: KeyError. Exception info: 'season'|
03/03/2021 07:42:24|WARNING |retry.api                       |HTTPSConnectionPool(host='www.podnapisi.net', port=443): Read timed out. (read timeout=10), retrying in 5 seconds...|
03/03/2021 07:42:39|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
03/03/2021 10:09:55|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 10:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 10:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 10:09:55|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
03/03/2021 10:42:19|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 10:52, because of: KeyError. Exception info: 'season'|
03/03/2021 10:42:21|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 10:52, because of: KeyError. Exception info: 'season'|
03/03/2021 10:42:34|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
03/03/2021 13:09:55|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 13:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 13:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 13:09:55|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
03/03/2021 13:42:11|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 13:52, because of: KeyError. Exception info: 'season'|
03/03/2021 13:42:13|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 13:52, because of: KeyError. Exception info: 'season'|
03/03/2021 13:42:19|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
03/03/2021 14:52:17|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 16:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 16:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 16:09:55|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
03/03/2021 16:42:02|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 16:52, because of: KeyError. Exception info: 'season'|
03/03/2021 16:42:04|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 16:52, because of: KeyError. Exception info: 'season'|
03/03/2021 16:42:31|WARNING |retry.api                       |HTTPSConnectionPool(host='www.podnapisi.net', port=443): Read timed out. (read timeout=10), retrying in 5 seconds...|
03/03/2021 16:44:12|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
03/03/2021 19:06:51|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 19:06:52|INFO    |root                            |Provider opensubtitles throttle count 1 of 5, waiting 5sec and trying again|
03/03/2021 19:06:55|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 19:16, because of: KeyError. Exception info: 'season'|
03/03/2021 19:06:57|INFO    |root                            |Provider opensubtitles throttle count 2 of 5, waiting 5sec and trying again|
03/03/2021 19:07:02|INFO    |root                            |Provider opensubtitles throttle count 3 of 5, waiting 5sec and trying again|
03/03/2021 19:07:08|INFO    |root                            |Provider opensubtitles throttle count 4 of 5, waiting 5sec and trying again|
03/03/2021 19:07:23|INFO    |root                            |Throttling opensubtitles for 20 minutes, until 21/03/03 19:27, because of: ServiceUnavailable. Exception info: '503'|
03/03/2021 19:07:24|INFO    |root                            |Throttling opensubtitles for 20 minutes, until 21/03/03 19:27, because of: ServiceUnavailable. Exception info: '503'|
03/03/2021 19:07:25|INFO    |root                            |Throttling opensubtitles for 20 minutes, until 21/03/03 19:27, because of: ServiceUnavailable. Exception info: '503'|
03/03/2021 19:07:26|INFO    |root                            |Throttling opensubtitles for 20 minutes, until 21/03/03 19:27, because of: ServiceUnavailable. Exception info: '503'|
03/03/2021 19:07:27|INFO    |root                            |Throttling opensubtitles for 20 minutes, until 21/03/03 19:27, because of: ServiceUnavailable. Exception info: '503'|
03/03/2021 19:07:29|INFO    |root                            |Throttling opensubtitles for 20 minutes, until 21/03/03 19:27, because of: ServiceUnavailable. Exception info: '503'|
03/03/2021 19:09:29|INFO    |root                            |BAZARR Finished searching for Subtitles to upgrade. Check History for more information.|
03/03/2021 19:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 19:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 19:09:55|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
03/03/2021 19:23:49|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 19:24:09|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 19:34, because of: KeyError. Exception info: 'season'|
03/03/2021 19:24:11|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 19:34, because of: KeyError. Exception info: 'season'|
03/03/2021 19:24:27|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
03/03/2021 19:25:27|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 19:25:27|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 19:25:27|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
03/03/2021 19:41:21|INFO    |root                            |Using opensubtitles again after 20 minutes, (disabled because: ServiceUnavailable)|
03/03/2021 19:41:21|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 19:42:04|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 19:52, because of: KeyError. Exception info: 'season'|
03/03/2021 19:42:11|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 19:52, because of: KeyError. Exception info: 'season'|
03/03/2021 19:42:23|WARNING |retry.api                       |HTTPSConnectionPool(host='www.podnapisi.net', port=443): Read timed out. (read timeout=10), retrying in 5 seconds...|
03/03/2021 19:42:40|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
03/03/2021 21:46:32|INFO    |root                            |Using soustitreseu again after 10 minutes, (disabled because: KeyError)|
03/03/2021 22:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 22:09:55|ERROR   |root                            |BAZARR Error trying to get video information for this file: /data/media/movies/The Avengers (2012)/The Avengers (2012) BR-DISK.iso|'Traceback (most recent call last):\n  File "/app/bazarr/bazarr/get_subtitle.py", line 62, in get_video\n    video = parse_video(path, hints=hints, providers=providers, dry_run=used_scene_name,\n  File "/app/bazarr/bazarr/../libs/subzero/video.py", line 61, in parse_video\n    return scan_video(fn, hints=hints, dont_use_actual_file=dry_run, providers=providers,\n  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 504, in scan_video\n    raise ValueError(\'%r is not a valid video extension\' % os.path.splitext(path)[1])\nValueError: \'.iso\' is not a valid video extension'|
03/03/2021 22:09:55|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
03/03/2021 22:41:47|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 22:51, because of: KeyError. Exception info: 'season'|
03/03/2021 22:41:54|INFO    |root                            |Throttling soustitreseu for 10 minutes, until 21/03/03 22:51, because of: KeyError. Exception info: 'season'|
03/03/2021 22:42:04|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
